pages/careers/open-application.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { Layout } from '../../layouts/article/layout';
+
+const Header = () => (
+  <section className="header-main">
+    <article className="content">
+      <h1>
+        {' '}
+        <span role="img" aria-label="check-mark">
+          📄✍️
+        </span>{' '}
+        Avoin hakemus
+      </h1>
+      <p>
+        <strong>Kenelle:</strong> motivoitunut Ohjelmistokehittäjä, UX Designer,
+        tai muu alan osaaja
+      </p>
+      <p>
+        <strong>Sijainti:</strong> Tampere
+      </p>
+      <p>
+        <strong>Palkka:</strong> neuvotellaan sopivaksi hakijan mukaan
+      </p>
+    </article>
+
+    <style jsx>{`
+      .header-main {
+        display: flex;
+      }
+      .content {
+        width: 60%;
+      }
+      .image {
+        display: flex;
+        align-items: center;
+        width: 40%;
+      }
+      @keyframes animate {
+        from {
+          transform: rotate(-60deg);
+        }
+        to {
+          transform: rotate(0deg);
+        }
+      }
+      .image img {
+        width: 100%;
+        transform: rotate(-60deg);
+        animation: 2s ease-out 0s 1 animate forwards;
+      }
+
+      @media (prefers-reduced-motion) {
+        .image img {
+          animation: none;
+        }
+      }
+
+      @media (max-width: 1150px) {
+        .image {
+          display: none;
+        }
+        .content {
+          width: auto;
+        }
+      }
+    `}</style>
+  </section>
+);
+
+const Page: React.FC = () => (
+  <Layout header={<Header />}>
+    <section className="content">
+      <article className="article">
+        <p>
+          Oletko jo hieman kokemusta kartuttanut web-kehittäjä, jota kiinnostaa
+          koodaamisen lisäksi myös ohjelmistokehitystyön pehmeämmät puolet?
+          Nautitko projektien speksaamisesta, prosessien kehittämisestä tai työn
+          ohjaamisesta oikeaan suuntaan? Saatko kiksejä, kun pystyt auttamaan
```

Comment:
kiksit kyllä kuulostaa ehkä vähän seksuaaliselta omaan korvaan :D en kyl tiiä millä korvais näin äkkiseltään 🤔

Reply:
:D laitan vaik et "Tuntuuko palkitsevalta"

Reply:
tai "tuntuuko siisiltä"